### PR TITLE
gmic 1.7.8 (new formula)

### PR DESCRIPTION
**Pull request by bartleusink**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Hi!
This PR adds a formula for [gmic](http://gmic.eu)

### CODE REVIEW

File: Formula/gmic.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  desc "Full-Featured Open-Source Framework for Image Processing"
+  homepage "https://gmic.eu/"
+  url "https://gmic.eu/files/source/gmic_1.7.8.tar.gz"
+  sha256 "3a2f32d79714239cfa56ecb10799b7d73362b4e2a852444bc24866272cd041a4"
+
+  depends_on "cmake" => :build
+  depends_on "jpeg" => :recommended
+  depends_on "libpng" => :recommended
+  depends_on "fftw" => :recommended
+  depends_on "opencv" => :optional
```
Comment (dunn) [inline Formula/gmic.rb]: This is from the Science tap, so it should be `depends_on "homebrew/science/opencv" => :optional`

File: Formula/gmic.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  depends_on "ffmpeg" => :optional
+  depends_on "libtiff" => :optional
+  depends_on "openexr" => :optional
+
+  def install
+    system "cmake", *std_cmake_args
+    system "make", "install"
+    man1.install "man/gmic.1.gz"
```
Comment (dunn) [inline Formula/gmic.rb]: Why isn't this done as part of the `make install`?

File: Formula/gmic.rb (hunk 1 above)
Comment (bartleusink) [inline Formula/gmic.rb]: Oops, fixed!

File: Formula/gmic.rb (hunk 2 above)
Comment (bartleusink) [inline Formula/gmic.rb]: I've left the manpage installation in for now but added a comment with the upstream issue.
Hopefully this line can be removed with the next version.

Is this OK for now?

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by frhgit** [comment]
Thank you for this formula. 
Do you know why it doesnt come with the gimp plug-in gmic_gimp as mention in the readme?

**Pull request closed without merging**